File: BOJ/C++/2440.cpp
#include <iostream>

int main()
{
	int count(0);
	std::cin >> count;
	for (int i = count; i > 0; --i)
	{
		for (int j = 0; j < i; ++j)
		{
			std::cout << '*';
		}
		std::cout << '\n';
	}
	return 0;
}

File: BOJ/rust/src/2440.rs
fn main() {
    let mut input_string = String::new();
    std::io::stdin().read_line(&mut input_string).expect("Wrong input");
    let mut count: i32 = input_string.trim().parse().expect("Not Number");
    
    loop {
        if count < 0
        {
            break;
        }
        
        let mut i = count;
        loop {
            if i < 1
            {
                break;
            }
            print!("*");
            i -= 1;
        }
        print!("\n");
        count -= 1;
    }
}
